impl/src/settings.rs

```diff
@@ -531,6 +531,45 @@ impl RawRazeSettings {
       || self.rust_rules_workspace_name.is_some()
       || self.vendor_dir.is_some()
   }
+
+  fn print_deprecation_warnings(&self) {
+    if self.incompatible_relative_workspace_path.is_some() {
+      eprintln!(
+        "WARNING: `[*.raze.incompatible_relative_workspace_path]` will soon be deprecated. Please \
+         update your project to use the `true` behavior."
```

Comment:
This warning should probably be more explicit about what the user should do. I looked up the field. Seems like the non-deprecated behavior is to have `workspace_path` be a valid bazel label, not a path relative to cargo.toml. Update the error to convey this?

Reply:
The non-deprecated behavior will use `workspace_path` to determine where the output is. Previously, users would put their `Cargo.toml` files in some path and then that path would be rendered into the files. However, if that path was wrong or you for some reason didn't want to move your `Cargo.toml` file to some other directory (ie. you want to maintain compatibility with Cargo), then you had to fumble around with `--output` and write a wrapper for running raze. 

The non-deprecated behavior will always place your files in the location of that label, making it the source of truth and easier to use in that you can specify this behavior in your toml files without needing to write a wrapper or change your project structure.

I can definitely update the wording

Reply:
Updated